infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationUtils.kt

```diff
@@ -261,50 +266,24 @@ fun <T, U> compareChange(
     propKey: PropKey,
     remark: PublicationChangeRemark? = null,
     enumLocalizationKey: String? = null,
-) =
-    if (predicate()) {
-        PublicationChange(
-            propKey,
-            value = ChangeValue(
-                oldValue = oldValue?.let(valueTransform),
-                newValue = newValue?.let(valueTransform),
-                localizationKey = enumLocalizationKey,
-            ),
-            remark,
-        )
-    } else null
-
-val MATH_POINT_TRANSLATION = "matemaattinen piste"
```

Comment:
`switchBaseTypeToProp()`-logiikkaa lukuunottamatta nämä kaikki käännökset tästä eteenpäin jäivät aika lailla turhiksi